app/javascript/oldjs/services/alerts_center_service.js

```diff
@@ -518,7 +518,7 @@ angular.module('ManageIQ').service('alertsCenterService', ['API', '$q', '$timeou
     newAlert.lastUpdate = newAlert.evaluated_on;
     newAlert.numComments = 0;
 
-    if (alertData.assignee) {
+    if (alertData.assignee !== undefined) {
       newAlert.assigned = true;
```

Comment:
you did this change?

Reply:
no, nothing here was me

Reply:
This is changed in https://github.com/ManageIQ/manageiq-ui-classic/pull/7726, not sure why again changes showing up, please check.